src/server/game/Handlers/QuestHandler.cpp

```diff
@@ -451,6 +451,13 @@ void WorldSession::HandleQuestLogRemoveQuest(WorldPackets::Quest::QuestLogRemove
             _player->RemoveActiveQuest(questId);
             _player->DespawnPersonalSummonsForQuest(questId);
 
+            bool updateVisibility = false;
```

Comment:
we may want this part in multiple places, similar to usages of `Player::DespawnPersonalSummonsForQuest`

Reply:
Fixed 👍